NEWS.md

```diff
@@ -4,6 +4,10 @@
 
 NOT RELEASED YET
 
+ * Fix generic Windows builds by setting rsync_EXPORTS in CMakeLists.txt,
```

Comment:
This is actually adding LIBRSYNC_EXPORT to all exported variables in librsync.h, not just "certain variables". So this would be a better description (assuming librsync_EXPORT definition is not required);

* Fix MSVC builds by adding missing LIBRSYNC_EXPORT to variables in librsync.h, and correctly install .dll files in the bin directory.

Reply:
Fixed.